src/containers/SearchContainer.jsx

```diff
@@ -28,31 +26,23 @@ export default class SearchContainer extends React.Component {
     };
   }
 
-  static getDerivedStateFromProps(nextProps) {
-    if (nextProps.searchQuery !== '' && !nextProps.isSearch) {
-      return {
-        searchQuery: '',
-      };
+  componentDidUpdate(prevProps) {
+    const { title: prevTitle } = prevProps;
+    const { title } = this.props;
+    if (prevTitle !== title) {
```

Comment:
I think it might be better to use `searchQuery` instead `title`.  ` if (prevSearchQuery !== searchQuery && !searchQuery) {`.